confluence-slack-integration/confluence-slack-server-integration-plugin/src/main/java/com/atlassian/confluence/plugins/slack/spacetochannel/api/notifications/DefaultAttachmentBuilder.java

```diff
@@ -79,21 +79,25 @@ public Attachment buildAttachment(final SpaceContentEntityObject content) {
     }
 
     private String calculateFooterTimestampText(final SpaceContentEntityObject content) {
+        final ConfluenceUser creator = content.getCreator();
         final String slackPublishedTimestamp = i18nResolver.getText(
                 "slack.activity.footer",
                 getSlackPrettyDate(content.getCreationDate().getTime()),
-                userLink(content.getCreator()));
+                userLink(creator));
 
         final int version = content.getVersion();
         if (version == 1 || content.getLastModifier() == null) {
             return slackPublishedTimestamp;
         } else {
+            final ConfluenceUser lastModifier = content.getLastModifier();
             final String slackUpdatedTimestamp = i18nResolver.getText(
                     "slack.activity.footer.modified",
                     getSlackPrettyDate(content.getLastModificationDate().getTime()),
-                    userLink(content.getLastModifier()));
+                    userLink(lastModifier));
 
-            boolean creatorIsLastModifier = content.getCreator().getKey().equals(content.getLastModifier().getKey());
+            final boolean creatorIsLastModifier = creator != null
```

Comment:
How about using this

final boolean creatorIsLastModifier = isCreatorIsLastModifier(content);

Reply:
Variable name may be indeed changed to follow Java naming conventions closer.
Extracting logic of the boolean value calculation to a separate method doesn't make sense though as it isn't used anywhere more.